spec/Section 2 -- Language.md

```diff
@@ -1247,22 +1247,22 @@ input type.
 
 Type : Name
 
-- Let {name} be the string value of {Name}
+- Let {name} be the string value of {Name}.
 - Let {type} be the type defined in the Schema named {name}
```

Comment:
Why no full stop on that_ line?

Reply:
That is a great question. I wonder how this escaped the editing...

Reply:
Ah; it's because this isn't detected as an algorithm, algorithms start `Foo(a, b, c):`, so this has to be manually addressed.